api/v1alpha1/flowcollector_types.go

```diff
@@ -164,6 +164,12 @@ type FlowCollectorFLP struct {
 	// HealthPort is a collector HTTP port in the Pod that exposes the health check API
 	HealthPort int32 `json:"healthPort,omitempty"`
 
+	//+kubebuilder:validation:Minimum=1
+	//+kubebuilder:validation:Maximum=65535
+	//+kubebuilder:default:=9102
```

Comment:
Default for prometheus is generally 9090, or if already used 9091 and so on , any special reason to use 9102 here?

Reply:
Fixed, changed to 9090

(not sure why 9102 was used by @KalmanMeth in FLP ... @KalmanMeth any info to add here ???)

Reply:
@eranra @jotak Probably I ported this from code that already used port 9090 for the regular prometheus resource monitor, so another port was used for network metrics prometheus connector. Nothing sacred about the port number.